docs/book/src/design/wallet-tee.md

```diff
@@ -0,0 +1,25 @@
+# Wallet TEE
+
+The Wallet TEE consists of a trusted SGX enclave, alongside an untrusted host service.
```

Comment:
Perhaps link an overview of SGX, like so
```suggestion
 The Wallet TEE consists of a trusted [Intel SGX enclave](https://www.intel.com/content/www/us/en/developer/tools/software-guard-extensions/overview.html), alongside an untrusted host service.
```

Reply:
We do link to this in the main overview.

My instinct would be to assume some general familiarity with Intel SGX once it gets into this level of detail (documenting individual system components), and to limit the links to general SGX introductory material more to the opening page, glossary, or a dedicated SGX / TEE background section?

Reply:
Oh, I missed that one.  Agreed, it is sufficient to link only the first occurence and to assume it understood thereafter.